### DISCUSSION TITLE
제페토 타입스크립트 문서에 대한 문의드립니다.

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#239] [5. Q&A]
안녕하세요

제페토 타입스크립트를 조금씩 사용해보면서 개발에 적응해가고 있는데요.

아무래도 정보가 너무 부족해서 어려움이 많은 것 같습니다

타입스크립트에 대한 API Docs는 혹시 진행되고 있을까요 ?? 아니면 진행예정인지도 궁금합니다. ㅠㅠ

**Comment by pda017** [comment]
api가 유니티에서 그대로 가져왔기때문에 일부 기능빼고는 거의 다 작동하는거 같아요.  유니티 api를 참고하시면 좋을거 같아요.
Resources 클래스는 좀 제대로 작동 안하더라구요.

**Reply by lhj5709** [reply]
제페토 타입스크립트 자체의 전체 인터페이스가 나오는 문서를 여쭤본겁니다,,ㅠㅠ
예를들면 ZEPETO.Multiplay 내부의 구성이요,, 공식 홈페이지에 있는 내용이 전부인지 궁금하구요

**Comment by hyeonjin-jo** [comment]
안녕하세요.

공식 가이드문서 이외에 class 별로 Property나 function에 대한 문서를 내부적으로 고려하고 있습니다.

확정일자를 안내드리긴 어려운점 양해부탁드립니다.

감사합니다.
